### PR TITLE
Fix building of docs with latest hugo version

### PR DESCRIPTION
**Pull request by lafriks**
Fixes latest CI failure because of using newer hugo version

### CODE REVIEW

**Comment by lunny** [comment]
LGTM

**Comment by lafriks** [comment]
~~@lunny review comments are not counted as LG-TM ;)~~ looks like are but not automatically

File: docs/scripts/trans-copy [hunk 1]
```diff
@@ -28,6 +28,8 @@ for SOURCE in $(find ${ROOT}/content -type f -iname *.en-us.md); do
     if [[ ! -f ${DEST} ]]; then
       echo "Creating fallback for ${DEST#${ROOT}/content/}"
       cp ${SOURCE} ${DEST}
+      sed -i.bak "s/en\-us/${LOCALE}/g" ${DEST}
+      rm ${DEST}.bak
```
Comment (thehowl) [inline docs/scripts/trans-copy:32]: why not a simple `sed -i` without `.bak` so that it modifies the file in place?

Comment (lafriks) [reply docs/scripts/trans-copy:32]: Because on macosx sometimes it has problems by directly replacing in same file

**Comment by thehowl** [comment]
alright then, LGTM.

**Pull request merged**